### ISSUE TITLE
sample notebook crashes: client stream closed

### ISSUE THREAD

**Issue by floswald** [fonsp/Pluto.jl#172]
hi,

i wanted to try out the basic.jl sample but i got this when trying to execute a new cell with `1+1` in it:

```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.4.2 (2020-05-23)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.4) pkg> build Pluto

julia> import Pluto

julia> Pluto.run()
Go to http://localhost:1234/ to start writing ~ have fun!

Press Ctrl+C in this terminal to stop Pluto

[ Info: Client glmmi8 stream closed.
┌ Warning: Response function to message of type changecell failed
└ @ Pluto ~/.julia/packages/Pluto/023PH/src/webserver/WebServer.jl:292
┌ Warning: Reading WebSocket client stream failed for unknown reason:
│   exception =
│    KeyError: key "code" not found
│    Stacktrace:
│     [1] getindex at ./dict.jl:477 [inlined]
│     [2] (::Pluto.var"#117#119")(::Dict{String,Any}, ::Pluto.Notebook, ::Pluto.Cell; initiator::Pluto.Initiator) at /Users/74097/.julia/packages/Pluto/023PH/src/webserver/Dynamic.jl:112
│     [3] process_ws_message(::Dict{String,Any}, ::HTTP.WebSockets.WebSocket{HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at /Users/74097/.julia/packages/Pluto/023PH/src/webserver/WebServer.jl:290
│     [4] (::Pluto.var"#72#76")(::HTTP.WebSockets.WebSocket{HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at /Users/74097/.julia/packages/Pluto/023PH/src/webserver/WebServer.jl:152
│     [5] upgrade(::Pluto.var"#72#76", ::HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}; binary::Bool) at /Users/74097/.julia/packages/HTTP/BOJmV/src/WebSockets.jl:146
│     [6] upgrade at /Users/74097/.julia/packages/HTTP/BOJmV/src/WebSockets.jl:129 [inlined]
│     [7] (::Pluto.var"#71#75")(::HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}) at /Users/74097/.julia/packages/Pluto/023PH/src/webserver/WebServer.jl:128
│     [8] handle at /Users/74097/.julia/packages/HTTP/BOJmV/src/Handlers.jl:269 [inlined]
│     [9] #4 at /Users/74097/.julia/packages/HTTP/BOJmV/src/Handlers.jl:345 [inlined]
│     [10] macro expansion at /Users/74097/.julia/packages/HTTP/BOJmV/src/Servers.jl:367 [inlined]
│     [11] (::HTTP.Servers.var"#13#14"{HTTP.Handlers.var"#4#5"{HTTP.Handlers.StreamHandlerFunction{Pluto.var"#71#75"}},HTTP.ConnectionPool.Transaction{Sockets.TCPSocket},HTTP.Streams.Stream{HTTP.Messages.Request,HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}})() at ./task.jl:358
└ @ Pluto ~/.julia/packages/Pluto/023PH/src/webserver/WebServer.jl:163

julia> versioninfo()
Julia Version 1.4.2
Commit 44fa15b150* (2020-05-23 18:35 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.7.0)
  CPU: Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, haswell)
Environment:
  JULIA_EDITOR = sublime

(@v1.4) pkg> st Pluto
Status `~/.julia/environments/v1.4/Project.toml`
  [c3e4b0f8] Pluto v0.9.5

```

I ran this in the lastest version of firefox, FWIW

**Comment by fonsp** [comment]
Could you try refreshing your browser tab?

**Comment by floswald** [comment]
erm, to be honest i just tried again after a while and it works fine? i don't understand. good if you found something!

**Comment by fonsp** [comment]
It looks like `Ctrl+Enter` works, but the play button doesn't 😬

**Comment by floswald** [comment]
oh yeah, that's it! :smile:

**Comment by fonsp** [comment]
Fixed in `v0.9.7`, you can update in about 30 minutes, when [this PR](https://github.com/JuliaRegistries/General/pull/16100) is merged
